package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Prowac",
+  "version": "0.1.0",
+  "description": "Progressive Web App Crawler",
+  "main": "gulpfile.babel.js",
+  "engines": {
+    "node": ">= 4"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mozilla/prowac.git"
+  },
+  "author": "Tim Abraldes",
+  "contributors": [
+    "Harald Kirschner <npm@digitarald.com> (http://digitarald.de/)",
+    "Piotr Zalewa <zaloon@gmail.com> (http://piotr.zalewa.info/)"
+  ],
+  "license": "MPL-2.0",
+  "bugs": {
+    "url": "https://github.com/mozilla/prowac/issues"
+  },
+  "homepage": "https://github.com/mozilla/prowac",
+  "dependencies": {
+    "babel-core": "^6.4.5",
+    "babel-eslint": "^4.1.8",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-es2015-loose": "^7.0.0",
+    "babel-preset-stage-0": "^6.3.13",
+    "eslint-config-airbnb": "^4.0.0",
+    "express": "^4.13.4",
+    "gulp": "^3.9.0",
+    "gulp-eslint": "^1.1.1",
+    "gulp-load-plugins": "^1.2.0",
+    "redis": "^2.4.2"
+  },
+  "devDependencies": {
+    "fetch-mock": "^4.1.0",
+    "intern": "^3.0.6"
```

Comment:
I think we'll actually end up using mocha for our test suite rather than intern. Let's remove these two dependencies until we actually have a test framework set up